pkg/build/node/node.go

```diff
@@ -496,19 +497,88 @@ func (c *BuildContext) prePullImages(dir, containerID string) error {
 		return err
 	}
 
-	// preload images into containerd
-	// TODO(bentheelder): we can skip the move and chown steps if we go directly to this
-	if err = inheritOutputAndRun(cmder.Command(
-		"bash", "-c",
-		`containerd & find /kind/images -name *.tar -print0 | xargs -0 -n 1 -P $(nproc) ctr --namespace=k8s.io images import --no-unpack && kill %1 && rm -rf /kind/images/*`,
-	)); err != nil {
-		log.Errorf("Image build Failed! Failed to load images into containerd %v", err)
+	// setup image importer
+	importer := newContainerdImporter(cmder)
+	if err := importer.Prepare(); err != nil {
+		log.Errorf("Image build Failed! Failed to prepare containerd to load images %v", err)
+		return err
+	}
+
+	// TODO: return this error?
```

Comment:
I'm not sure how much we care, and this is hardly the messiest thing in this file :/